spring-boot-project/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -2757,6 +2757,20 @@
       "name": "spring.session.redis.flush-mode",
       "defaultValue": "on-save"
     },
+    {
+      "name": "spring.session.redis.repository",
```

Comment:
I wasn't 100% sure how to name this property, so I initially went with the simple option. Alternatively, this could maybe be named `spring.session.redis.repository-type`.